pyspectrafuse/common/parquet_utils.py

```
@@ -9,23 +10,34 @@ def __init__(self, path: str):
 
     def get_item_info(self) -> str:
-        """Extract item identifier from parquet path.
-        
+        """Extract item identifier (e.g., PXD008467) from parquet path.
+
         Returns:
-            Item identifier (typically dataset ID like PXD008467)
+            Item identifier (letters + digits prefix from the last path part)
         """
-        return self.path_obj.parts[-1].split('-')[0]
+        # 获取路径的最后一个部分（文件名或目录名）
```

Ruff (0.15.6)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)